src/lib/__tests__/InstantSearch-test.js

```diff
@@ -185,11 +171,8 @@ describe('InstantSearch lifecycle', () => {
         });
 
         it('calls widget.render({results, state, helper, templatesConfig, instantSearchInstance})', () => {
-          expect(widget.render.calledOnce).toBe(
-            true,
-            'widget.render called once'
-          );
-          expect(widget.render.args[0]).toMatchSnapshot();
+          expect(widget.render).toHaveBeenCalledTimes(1);
+          expect(widget.render.mock.calls[0][0]).toMatchSnapshot();
```

Comment:
- before:`expect(widget.render.args[0]).toMatchSnapshot();` is checking all args of the first call
- after:`expect(widget.render.mock.calls[0][0]).toMatchSnapshot();` is checking the first arg of the first call
I'm just bringing this up in case it's unwanted.

Reply:
Thanks for spotting that!